src/box/session.h

```diff
@@ -375,6 +375,73 @@ generic_session_fd(struct session *session);
 int64_t
 generic_session_sync(struct session *session);
 
+/**
+ * Get the unique identifier (ID) for the current session.
+ *
+ * \retval session ID
```

Comment:
As you probably see, there is no a single place in `session.h` which uses `\` instead of `@` for doxygen comments. Please, conform.